### DISCUSSION TITLE
Option & Develop Keys?

### DISCUSSION THREAD

**Discussion by jcsnyc1** [rsjaffe/MIDI2LR#643] [Q&A] [answered]
Hi - Great tool! Is there a way to combine OPTION with some of the develop sliders like BLACKS, SHADOWS, HIGHLIGHTS or WHITES? When using the mouse and option, you can see an overlay. Not sure how to do this with Midi Controller knob and Option Key....

**Comment by rsjaffe** [comment, marked as answer]
Option doesn't work with MIDI controller due to Lightroom bug. See [FAQ 6](https://github.com/rsjaffe/MIDI2LR/wiki#faqknown-issues).
